### PR TITLE
Feature/rules instore

### PR DESCRIPTION
**Pull request by biancameira12**
#### What is the purpose of this pull request?
Change document rules for instore

#### What problem is this solving?
Mexican document is not required anymore and brasilian document mask

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/rules/BRA.js [hunk 1]
```diff
@@ -34,40 +34,51 @@ export default {
...
-        return secondDigit === cleanValue.charAt(10)
+      validate: cpf => {
+        if (!cpf) {
+          return false
+        }
+
+        let sum = 0
+        let remainder
+
+        cpf = !cpf || typeof cpf !== 'string' ? '' : cpf.replace(/[^\d]+/g, '')
```
Comment (ArthurSampaio) [inline react/rules/BRA.js:44]: Do not mutate you variable. 
And, if cpf isnt of string type, what you do?  Try do this verification on L38

File: react/rules/BRA.js [hunk 2]
```diff
@@ -34,40 +34,51 @@ export default {
...
+          cpf === '77777777777' ||
+          cpf === '88888888888' ||
+          cpf === '99999999999'
+        ) {
+          return false
+        }
+
+        for (let i = 1; i <= 9; i++) {
+          sum += parseInt(cpf.substring(i - 1, i), 10) * (11 - i)
+        }
```
Comment (ArthurSampaio) [inline react/rules/BRA.js]: try use reducer https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

File: react/rules/MEX.js [hunk 3]
```diff
@@ -39,7 +39,6 @@ export default {
       name: 'document',
       maxLength: 50,
       label: 'MEX_rfc',
-      required: true,
```
Comment (ArthurSampaio) [inline react/rules/MEX.js:42]: are you sure?

File: react/rules/BRA.js [hunk 4]
```diff
@@ -34,40 +34,51 @@ export default {
...
+
+        remainder = (sum * 10) % 11
+
+        if (remainder === 10 || remainder === 11) remainder = 0
+        if (remainder !== parseInt(cpf.substring(9, 10), 10)) return false
+
+        sum = 0
```
Comment (ArthurSampaio) [inline react/rules/BRA.js]: every time that you had the needed to re-assign a variable its a indication that you should think in encapsulate the code above this re-assign

Comment (biancameira12) [reply react/rules/BRA.js]: On my new commit I didn't re-assign that variable, so it shouldn't be a problem. But I didn't understand what you mean by 'encapsulate the code'

File: react/rules/MEX.js (hunk 3 above)
Comment (biancameira12) [reply react/rules/MEX.js:42]: Yes! Even has a test for it

**Pull request merged**